server/views/pages/goal/add-step/index.njk

```diff
@@ -22,11 +20,9 @@
     }) }}
   {% endif %}
 
-  {% include "../../../partials/prisonerBanner.njk" %}
-
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l">What are the steps to help them achieve their goal?</h1>
+      <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
```

Comment:
As above, I'm concerned about the implicit coupling between the page title and the question text 🤔

Reply:
(though I've just seen this is the h1, not a question text .... I'm still not sure though .... 🤔 )

Reply:
See above comment